zbench.zig

```diff
@@ -323,9 +378,10 @@ pub fn getSystemInfo() !platform.OsInfo {
 pub const Result = struct {
     name: []const u8,
     readings: Readings,
+    total_duration_ns: u64,
```

Comment:
Yeah you are right. Currently I have not found a clean way to handle this. When I tried to move it, I end up with a segfault. Need to have another look here.

Reply:
Where you're defining an array of `Result`s in `run()`, you could have an array `total_durations: []u64` instead and append the `total_duration` which you've calculated to that rather than make a new `Result` from the data in the `Result` you get in the `switch`'s `.result` clause. Then rather than pass a slice of `Result` values to `printSummary` you can just pass the `total_durations`, and to get the name for each one you use the index to look it up in the benchmarks given, just like you're doing for the `fastest_index`. Does that make sense?